packages/theming/theming-utils/src/mapPipelineToTheme.ios.ts

```diff
@@ -1,134 +1,116 @@
 import { AliasColorTokens, Variants, VariantValue } from '@fluentui-react-native/theme-types';
 import { FontStyleTokens } from '@fluentui-react-native/tokens';
 
+// API that translates tokens coming for android to Theme color values.
```

Comment:
nit n rename to iOS

Reply:
Updated. I thought some parts of the original comment were redundant so I replaced it/the Android comment with something that made more sense to me - happy to update it again if you think I removed some important information